src/store/machine/modelEnums.js

```diff
@@ -177,6 +177,7 @@ export const MoveShapingType = {
 	ZVD: 'ZVD',
 	ZVDD: 'ZVDD',
 	EI2: 'EI2',
+	EI3: 'EI3',
 	DAA: 'DAA'
```

Comment:
Please remove DAA again, it isn't supported any more in v3.4-b2. Also EI3 appears to be missing in the compiled plugin.

Reply:
should be part of DWC release as it's machine dependent and not a plugin dependency. created another PR to fix this issue. Check

https://github.com/Duet3D/DuetWebControl/pull/380

Reply:
Right, thanks for the note. I am afraid that means you have to hard-code that enum into the plugin for now.

Reply:
fixed in a seperate branch as it's just a temporary fix until next dwc is released.